app/assets/stylesheets/admin/global/_base.sass

```diff
@@ -16,6 +21,18 @@ div.container
   span
     @extend %visually-hidden
 
+thead, h1, h2, h3, h4, h5, h6
+  color: $heading-color
+
+thead, h1, h2, h3
+  font-family: $heading-fonts
+
+h1, h2, h3, h4, h5, h6
```

Comment:
Ultimately, there is no difference, the compiled output will be the same.

The difference here is that I just shuffled files around and didn't spend any time formatting the actual code.
It's just a visual thing and in CCIWA I had a [Sass Linter](https://github.com/sasstools/sass-lint) running that was complaining about [the format](https://github.com/sasstools/sass-lint/blob/develop/docs/rules/single-line-per-selector.md).

My goal is that eventually all files using or in Rails template are formatted the same. Can't spend that much time updating it all though. So whenever I work on a piece and have extra time, I will convert the files I am touching.

In this case, all the CSS files will be changed again later on when we have the gem, so I didn't spend any time formatting yet.